handlers/zipkin_test.go

```diff
@@ -1,6 +1,7 @@
 package handlers_test
 
 import (
+	"fmt"
```

Comment:
Looks like this import can be dropped?

Reply:
@stijnvetstijnvet it looks like this is still in the zipkin_test.go but resolved in zipkin.go?